store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -98,12 +98,12 @@ private String cleanMalformedHtml(String str) throws UnsupportedEncodingExceptio
             Tidy tidy = new Tidy();
             tidy.setInputEncoding("UTF-8");
             tidy.setOutputEncoding("UTF-8");
-            tidy.setPrintBodyOnly(true);
+            tidy.setPrintBodyOnly(false);
             tidy.setXHTML(true);
             ByteArrayInputStream inputStream = new ByteArrayInputStream(str.getBytes("UTF-8"));
             ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
             tidy.parseDOM(inputStream, outputStream);
-            String outStream = outputStream.toString("UTF-8");
+            String outStream = outputStream.toString("UTF-8").trim();
```

Comment:
this should not trim() if outputStream.toString("UTF-8")=null then it will be NPE. 

you want to trim() to check blank String which can be handled  if condition next line.